calculators/_memoria_rating.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+$GLOBALS['rating_calculators']['_memoria_rating'] = [
+	'description' => '
+		by <a href="/player/125980">Memoria</a>: Rating with per-matchup rating calculation, v1
+		
+		-Only for 1v1, 2v2, and 3v3.
+  		-Games under 4 minutes don\'t count.
+		-Just like in the main rating, people start at 2000 rating,
+		 4000 rating difference = 90% win prediction, max 100 rating per match
+  		-Rating prediction is calculated for each matchup (1v1),
+		 then it\'s combined into a final prediction 
```

Comment:
I suggest spending more time on this text, make it more beautiful and consistent:

- add spaces after "-", right now it looks like this, compared to level rating for example
![image](https://github.com/dima-stefantsov/dsr_ratings/assets/1370909/1a7c0aaf-92a7-4ca2-967b-a80ba2d4cd35)

- probably get rid of linebreak before "4000", why is it a newline if it's not a new bulletpoint? we are doing things for all kinds of device sizes, including something like
![image](https://github.com/dima-stefantsov/dsr_ratings/assets/1370909/e3134103-acf0-43c5-9db8-9bb7896dc710)
, therefore you want to have semantic linebreaks, not your grandpa-linebreaks where he thought it would look good on his specific device.

- I recommend getting rid of capital letters and periods in the end, to reduce the visual clutter. But if you insist, then at least add periods consistently, right now it's 50/50.

- a comment of "just like the main rating" may become obsolete with time, you generally don't want to write something that may become wrong without you even participating in it. Who knows if I maybe decide to make default 3000 tomorrow. Up to you.

- since you are referencing "main" rating, you may want to follow default "is game accepted" logic, with it being 30 seconds (shorter games will not appear in database at all). This is totally up to you, if your vision requires 4min filter, so be it!